spec/requests/categories_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'swagger_helper'
+
+RSpec.describe 'categories', type: :request do
+  path '/{locale}/categories' do
```

Comment:
The same. Or remove this file, or correct all data inside.

Reply:
Done - the file is removed